lib/membrane_rtc_engine/engine.ex

```diff
@@ -1083,6 +1110,13 @@ defmodule Membrane.RTC.Engine do
     |> Map.get(track_id)
   end
 
+  defp endpoint_ref(endpoint_id) do
+    Membrane.Child.ref(
+      {:endpoint, endpoint_id},
+      group: endpoint_id
+    )
```

Comment:
couldn't it be

```suggestion
    Membrane.Child.ref(:endpoint, group: endpoint_id)
```

?